openlibrary/plugins/openlibrary/js/index.js

```diff
@@ -27,7 +27,7 @@ import * as Browser from './Browser';
 import { commify } from './python';
 import { Subject, urlencode, slice } from './subjects';
 import Template from './template.js';
-// Add $.fn.focusNextInputField, $.fn.ol_confirm_dialog
+// Add $.fn.focusNextInputField
```

Comment:
Rather than doing this I've made it an explicit return value. Ideally index.js should define all the globals rather than requiring a module. 

Likewise, on the long term $.fn.focusNextInputField should be set inside this file.